src/components/Confirm/index.js

```diff
@@ -38,32 +38,25 @@ const Previews = ({capture, step, retakeAction, confirmAction} ) =>  {
 
 const getCapture = (captures) => captures[0]
 
-class Confirm extends Component {
-  render () {
-    const {
+const Confirm = ({
       step,
       method,
-      documentCaptures,
-      faceCaptures,
+      validCaptures,
       actions: {
         deleteCaptures,
         confirmCapture
       }
-    } = this.props
+    }) => {
 
-    const captures = {
-      'document': documentCaptures,
-      'face': faceCaptures
-    }[method]
-    const capture = getCapture(captures)
+  const captures = validCaptures[method]
```

Comment:
I always prefer going for a naming convention of `x` and `xList` or something, when I have both

Otherwise you have to spot the difference between `x` and `xs`, and it just ends up being confusing